driver-sync/src/test/functional/com/mongodb/client/AbstractClientSideEncryptionTest.java

```diff
@@ -249,7 +249,7 @@ public void setUp() {
                 .build(), commandListener);
 
         database = getDatabase(databaseName);
-        helper = new JsonPoweredCrudTestHelper(description, database, database.getCollection("default", BsonDocument.class));
+        helper = new JsonPoweredCrudTestHelper(description, database, database.getCollection(collectionName, BsonDocument.class));
```

Comment:
I'm unsure why this is the first test that exposed this.  Do you know?

Reply:
Its just the first test in `/client-side-encryption/` to not use default for the collection.